lib/palindrome_permutation.rb

```diff
@@ -1,4 +1,16 @@
 
 def palindrome_permutation?(string)
-  
+  hash_table = {}
```

Comment:
As with the other two methods, try to think of a variable name that is more descriptive of how the hash is used and what is stored in it.